src/Microsoft.AspNetCore.Mvc.Cors/CorsAuthorizationFilter.cs

```diff
@@ -20,16 +18,32 @@ public class CorsAuthorizationFilter : ICorsAuthorizationFilter
     {
         private readonly ICorsService _corsService;
         private readonly ICorsPolicyProvider _corsPolicyProvider;
+        private readonly ILogger _logger;
 
         /// <summary>
         /// Creates a new instance of <see cref="CorsAuthorizationFilter"/>.
         /// </summary>
         /// <param name="corsService">The <see cref="ICorsService"/>.</param>
         /// <param name="policyProvider">The <see cref="ICorsPolicyProvider"/>.</param>
         public CorsAuthorizationFilter(ICorsService corsService, ICorsPolicyProvider policyProvider)
+            : this(corsService, policyProvider, loggerFactory: null)
```

Comment:
Is the null logger factory public? Can we use that instead as a sentinel? It's much nicer to not have to deal with a null logger instance